### DISCUSSION TITLE
LDAP issue

### DISCUSSION THREAD

**Discussion by rakeshsujan** [netbox-community/netbox#17513] [General]
Hello,

We are trying to configure the LDAP in our Netbox server. We have done all the configuration and the services are running without any issue but when we try to access the Netbox web URL I am getting the below error.

Can any one help me on this issue.

![image](https://github.com/user-attachments/assets/0e69616c-1eee-425f-9b05-2f502fc10552)

**Comment by candlerb** [comment]
> We have done all the configuration

It looks like you may have omitted [this step](https://netboxlabs.com/docs/netbox/en/stable/installation/6-ldap/#install-django-auth-ldap)

```
source /opt/netbox/venv/bin/activate
pip3 install django-auth-ldap
sudo sh -c "echo 'django-auth-ldap' >> /opt/netbox/local_requirements.txt"
```

Personally, I prefer to combine the first two lines into this one:

```
/opt/netbox/venv/bin/pip3 install django-auth-ldap
```

You can check whether it has been installed:

```
/opt/netbox/venv/bin/pip3 freeze | grep -i ldap
```

**Comment by rakeshsujan** [comment]
I have followed the steps but same issue find the below output for your reference 

 /opt/netbox/venv/bin/pip3 freeze | grep -i ldap
django-auth-ldap==4.8.0
python-ldap==3.4.4

**Reply by candlerb** [reply]
Can you get a backtrace from that error screen? Set DEBUG=True in configuration.py and restart.
